### DISCUSSION TITLE
Frequency List

### DISCUSSION THREAD

**Discussion by Lyhcan** [arianneorpilla/jidoujisho#234] [Q&A] [answered]
hi, i have a frequency list that i use on the migaku extension and was wondering if there is a way to make it work on jidoujisho? it's a json file, i tried to import it with no success.
The migaku frequency lists works like that: The first word in the list is "THE" and its frequency is 1 because its the first word, so if i search THE in the dictionary it will show me 1, etc.. )
! [Screenshot 2023-05-28 234337](https://github.com/lrorpilla/jidoujisho/assets/132207809/7f5c33fc-cd5a-427e-a8c3-7cc03c76de67)

**Comment by Lyhcan** [comment]
in case it's a simple formating issue i would appreciate if u could help me formatting it or give me ideas <3
[Corpus English Frequency List.zip](https://github.com/lrorpilla/jidoujisho/files/11587842/Corpus.English.Frequency.List.zip)

**Comment by arianneorpilla** [comment, marked as answer]
Answered in #235.
